--- golang/1.8/alpine3.6/Dockerfile	2017-06-18 06:06:16.372901765 -0500
+++ golang/1.8/alpine3.6/Dockerfile.rpi	2017-06-18 09:29:33.570419425 -0500
@@ -1,4 +1,4 @@
-FROM alpine:3.6
+FROM whw3/alpine:latest
 
 RUN apk add --no-cache ca-certificates
 
--- golang/1.8/Dockerfile	2017-06-18 06:06:16.372901765 -0500
+++ golang/1.8/Dockerfile.rpi	2017-06-18 09:34:29.687936062 -0500
@@ -1,7 +1,7 @@
-FROM buildpack-deps:jessie-scm
-
+FROM resin/rpi-raspian
+RUN apt-get update && apt-get upgrade && apt-get install -y wget
 # gcc for cgo
-RUN apt-get update && apt-get install -y --no-install-recommends \
+RUN apt-get install -y --no-install-recommends \
 		g++ \
 		gcc \
 		libc6-dev \
